coretex/cli/modules/node.py

```diff
@@ -355,8 +377,8 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
         config["cpuCount"] = promptCpu(config, cpuLimit)
         config["nodeRam"] = promptRam(config, ramLimit)
+        config["nodeSwap"] = promptSwap(config.get("nodeRam", 0), swapLimit)
```

Comment:
nodeRam must exist at this point, it is previously configured with `promptRam` function